applications/MeshMovingApplication/custom_utilities/explicit_mesh_moving_utilities.cpp

```diff
@@ -237,7 +246,9 @@ namespace Kratos
 
     inline double ExplicitMeshMovingUtilities::ComputeKernelValue(const double NormalisedDistance){
         // Epanechnikov (parabolic) kernel function
-        return (std::abs(NormalisedDistance) <= 1.0) ? std::abs((3.0/4.0)*(1.0-std::pow(NormalisedDistance,2))) : 0.0;
+        // return (std::abs(NormalisedDistance) <= 1.0) ? std::abs((3.0/4.0)*(1.0-std::pow(NormalisedDistance,2))) : 0.0;
```

Comment:
pls remove the comment

Reply:
ah ok
could you add a comment abt this then?

Or why not make the selection optional? Then you don't have to recompile during testing

Reply:
Completely agree. But I don't want to have an if statement for each found point. Thus, I was thinking on defining a nested `Kernel` class for each kernel type inside the current one. Since this implies a further development out of the scope of this PR, I'd leave it for a future PR.